test/Microsoft.ServiceFabric.Services.Tests/Remoting/FabricTransport/Runtime/FabricTransportRemotingListenerSettingsTest.cs

```diff
@@ -0,0 +1,98 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Microsoft.ServiceFabric.Services.Remoting.FabricTransport.Runtime;
+using Microsoft.ServiceFabric.Services.Remoting.V2.Runtime;
+using Xunit;
+
+namespace Microsoft.ServiceFabric.Services
+{
+    public abstract class FabricTransportRemotingListenerSettingsTest
+    {
+        public FabricTransportRemotingListenerSettingsTest()
```

Comment:
this is odd. An abstract class shouldn't need a public constructor, particularly an empty one.